packages/terra-clinical-detail-view/CHANGELOG.md

```diff
@@ -3,7 +3,11 @@
 ## Unreleased
 * Changed
   * Updated the component examples.
-  
+
+* Added
```

Comment:
nit: from Vinay's comment on my PR, I think these should be in the Added section on line 11